apps/model-ad/app/src/config/config.json

```diff
@@ -1,9 +1,8 @@
 {
   "apiDocsUrl": "http://localhost:8000/api-docs",
-  "appVersion": "1.0.0-beta",
+  "appVersion": "edge",
   "csrApiUrl": "http://localhost:8000/api/v1",
-  "dataUpdatedOn": "yyyy-mm-dd",
-  "environment": "development",
   "googleTagManagerId": "",
-  "ssrApiUrl": "http://model-ad-api:3333/v1"
+  "ssrApiUrl": "http://model-ad-api:3333/v1",
+  "tagName": "edge"
```

Comment:
What is the purpose of this property? What resource is being tagged? Consider changing the property name to be more descriptive.